### PR TITLE
Adding WiFi dockerfile

### PR DESCRIPTION
**Pull request by stevesec**
# Description

> A description of your PR, what it brings or corrects. Don't forget to configure your PR to the dev branch (cf. https://exegol.readthedocs.io/en/latest/community/contributors.html)

- Added a WiFi.dockerfile in the root directory
- Added [Pretender](https://github.com/RedTeamPentesting/pretender) to the package_ad.sh file

# Related issues

> If your PR responds to an issue for a bug fix or feature request, make sure to includes references to the issues (e.g. "fixes #xxxx").

# Point of attention

> Things you are not sure about that deserve special attention if you have doubts or questions.

### CODE REVIEW

File: sources/install/package_ad.sh
Comment (ShutdownRepo) [inline sources/install/package_ad.sh:1]: This is a nice addition, but in my opinion it should be added in a separate pull request.
Also, regarding the review of that particular addition:
- you forgot to call the function at the bottom of the file in the package function
- why not install the tool via `go install -v $URL`?
- you used `add-history` and `add-aliases` but the corresponding files do not exist
- in `add-to-list`, the description of the tool should be changed to something like "an mitm tool for helping with relay attacks" as it doesn't only do DHCPv6

File: .github/workflows/entrypoint_preprod_wifi.yml
Comment (ShutdownRepo) [inline .github/workflows/entrypoint_preprod_wifi.yml:1]: This file will not be needed as the wifi image will not be built along official images for now as indicated in #288
The dockerfile will allow users to build their local image easily

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]
Thank you for the pull request!
The dockerfile seems good to me, but I have some change requests, see the comments below

File: .github/workflows/entrypoint_preprod_wifi.yml
Comment (stevesec) [reply .github/workflows/entrypoint_preprod_wifi.yml:1]: Understood. Thank you!

File: sources/install/package_ad.sh
Comment (stevesec) [reply sources/install/package_ad.sh:1]: Ah understood! I will make the necessary changes!

Comment (stevesec) [reply sources/install/package_ad.sh:1]: Updated in 141c33243193935f922b1e7a2861391ec5884b15

Comment (ShutdownRepo) [reply sources/install/package_ad.sh:1]: The addition of pretender should be made in a specific, separate, Pull Request

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]

File: sources/install/package_ad.sh
Comment (ShutdownRepo) [reply sources/install/package_ad.sh:1]: Indeed, #294 is created, but the changes still show here, please remove the pretender changes from here and only keep `wifi.dockerfile`

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**